draft-ietf-jsonpath-base.md

```diff
@@ -284,7 +286,10 @@ stable JSON query language.
 
 ## JSON Values
 
-The JSON value the JSONPath query is applied to is, by definition, a valid JSON value.
+The JSON value a JSONPath query is applied to is, by definition, a
+valid JSON value. A JSON value is usually constructed by parsing
```

Comment:
```suggestion
valid JSON value. A JSON value is often constructed by parsing
```

Reply:
Proposal not good?

Reply:
Yes, sorry, I thought I'd already adopted that suggestion.